cmd/eskip/readclient.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/eskipfile"
+	etcdclient "github.com/zalando/skipper/etcd"
+	innkeeperclient "github.com/zalando/skipper/innkeeper"
+	"io"
+	"io/ioutil"
+	"os"
+)
+
+type ReadClient interface {
+	LoadAndParseAll() ([]*eskip.RouteInfo, error)
+
+	LoadUpdate() (eskip.RouteList, []string, error)
```

Comment:
ups... I forgot it here... my initial intention was to completely replace the DataClient with a ReadClient and a WriteClient. That's also the reason for making them public :)